include/mxnet/op_attr_types.h

```diff
@@ -219,6 +219,15 @@ using FStatefulComputeEx = std::function<void (const OpStatePtr& state,
  */
 using FResourceRequest = std::function<
   std::vector<ResourceRequest> (const NodeAttrs& n)>;
+/*!
+ * \brief The resource request from the operator for FComputeEx.
```

Comment:
The resource request from the operator. (remove "for FComputeEx")
An operator could register ResourceRequestEx, or ResourceRequest, or neither.